faculty_cli/cli.py

```diff
@@ -1239,3 +1240,15 @@ def ls(project, path):
             click.echo("/project{}/".format(item.path))
         else:
             click.echo("/project{}".format(item.path))
+
+
+@cli.command()
+@click.argument("shell", type=click.Choice(["bash", "zsh", "fish"]))
+def completion(shell):
+    """Generate auto-completion scripts for faculty_cli."""
```

Comment:
Separately, have you considered putting the help of the README in https://docs.faculty.ai/user-guide/command_line_interface.html, where other CLI docs are currently?